cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -361,6 +362,8 @@ func (command *Evaluate) Execute(args []string) (err error) {
 	switch command.Runtime {
 	case "local":
 		return command.localRuntime(evaluationContext)
+	case "docker":
+		return command.dockerRuntime(evaluationContext)
```

Comment:
This is great! Two things that come to mind:

- [ ] we automatically install i.e. `ollama` and `symflower` so should we automatically install `docker` as well?
  - I think ultimately: yes - but that can be a follow up
- [ ] we should add tests for this that run symflower in a docker container on the golang/plain (I guess we have to manually install docker in the CI for now)

Reply:
Installing docker is not as trivial as install symflower or ollama, alone the Network stack problem that is happening with some configurations. 
Forcing to install docker/ollama on a user is something I do not like. If I only want to run openrouter models I don't need anything.
We should provide the right errors + maybe ask if the user wants that we install some stuff (except symflower as it is a hard dependency) 

Also "running" docker commands inside github actions isn't easy by itself, sure there is https://github.com/addnab/docker-run-action but that would run a job inside a container but we want to test our code which uses the "docker" shell command to run a container which is totally different.

Reply:
Follow up so we know this is untested for now.

Reply:
Don't get what you really want? I run it already multiple times as I had to verify it is working. The only thing you could do is checking out the branch and run it yourself! That would be a successful test.

Here is the command I used for testing:
```
make install && eval-dev-quality evaluate --runtime docker --result-path ./docker-test --runs 5 --model symflower/symbolic-execution --model symflower/symbolic-execution --model symflower/symbolic-execution --repository golang/plain
```

Reply:
Also here is the follow up issue: https://github.com/symflower/eval-dev-quality/issues/224